sds_data_manager/stacks/ialirt_processing_stack.py

```diff
@@ -36,107 +42,206 @@ def __init__(
             VPC into which to put the resources that require networking.
         repo : ecr.Repository
             ECR repository containing the Docker image.
-        instance_type : str
-            The EC2 instance type.
+        processing_name : str
+            Name of the processing stack.
+        ialirt_ports : list[int]
+            List of ports to for incoming traffic.
+        container_port : int
+            Port to be used by the container.
```

Comment:
Why is this one only a single `int` whereas above you have `list[int]`. I thought we wanted a one-to-one mapping of ports, not a `many-to-one` relationship?

Below I thought we could just say `for port in ialirt_ports: container_port=port, host_port=port`

Reply:
We want all of the traffic from the primary ports to be directed to the first container. And then all of the traffic from the secondary ports to be directed to the second container. The Dockerfile only exposes one port:

# Make port 8080 available to the world outside this container
EXPOSE 8080

Therefore any of the primary ports will be directed to port 8080. It is probably possible to expose the Docker container to other ports and have a one-to-one ratio, but then every time you add another port you have to create a new Docker image.